lua/litee/gh/pr/diff_view.lua

```diff
@@ -449,6 +449,8 @@ function M.open_diffsplit(commit, file, thread, compare_base)
     vim.cmd("vert diffsplit " .. diff_buffer_name)
     vim.cmd("q")
     vim.cmd("vert diffsplit " .. diff_buffer_name)
+    -- make the tmp buffer ephemeral
+    vim.api.nvim_set_option_value('bufhidden', 'delete', { buf = 0 })
```

Comment:
What happens if you want to move away from a duff buffer? Can you no longer get back to it?

Reply:
According to `:h bufhidden`:

```
'bufhidden' 'bh'	string	(default "")
			local to buffer  |local-noglobal|
	This option specifies what happens when a buffer is no longer
	displayed in a window:
	  <empty>	follow the global 'hidden' option
	  hide		hide the buffer (don't unload it), even if 'hidden' is
			not set
	  unload	unload the buffer, even if 'hidden' is set; the
			|:hide| command will also unload the buffer
	  delete	delete the buffer from the buffer list, even if
			'hidden' is set; the |:hide| command will also delete
			the buffer, making it behave like |:bdelete|
	  wipe		wipe the buffer from the buffer list, even if
			'hidden' is set; the |:hide| command will also wipe
			out the buffer, making it behave like |:bwipeout|
```

So the buffer is automatically deleted when no longer in a displayed window. It matches what I expect from a diff buffer. I took inspiration from how (fugitive)[https://github.com/tpope/vim-fugitive] does it.

If you need to reopen the buffer after quitting it, my workflow is often reopening the diff from the PR panel, so it makes sense to me.

Is it something you would find problematic ?

Reply:
Seems OK to me.